app/javascript/src/utilities/ajax.js

```diff
@@ -4,23 +4,23 @@ export type Method = 'GET' | 'POST' | 'DELETE'
 
 type FetchFunction = (url: string, method: Method, body?: URLSearchParams) => Promise<Response>
 
-const _ajax = (headers: Object) => {
+const { signal, abort }: AbortController = new AbortController()
```

Comment:
Is this being used anywhere, or it's for the future?

Reply:
The idea is to allow cancelling any request from our React components. However, I've been playing with it in the New Application Form PR and it doesn't quite work so I'll let each component have their own `AbortController`.